Niyonizeye/AnnounceIT
# Ft User Can Create announcement 170775268

### What does this PR do
 -  User  should create announcement
### Description of task to be completed
- Added Html
- Added Style
### How should this be manually tested
- Clone git repository
- https://github.com/Niyonizeye/AnnounceIT.git
- Open index.html in browser
- see the right of home page
- Click on login
### Any background context you want to provide?
- N/A

### What are relevant pivotal tracker stories ?
- https://www.pivotaltracker.com/story/show/170775268

### Screenshots (if appropriate)
![announce](https://user-images.githubusercontent.com/45454232/72837824-d0db9080-3c97-11ea-8f4c-65e803681981.PNG)
